plugins/parsers/csv/parser.go

```diff
@@ -169,6 +171,12 @@ func parseCSV(p *Parser, r io.Reader) ([]telegraf.Metric, error) {
 	for _, record := range table {
 		m, err := p.parseRecord(record)
 		if err != nil {
+			if p.SkipErrors {
+				if p.Log != nil {
+					p.Log.Infof("[parsers.csv] Malformed csv line: %s", err.Error())
```

Comment:
Stil not sure if this is a good format, could you paste an example of how the total error message in the log will look like?

At least it needs to be changed to 
```suggestion
					p.Log.Errorf("[parsers.csv] Malformed csv line: %w", err)
```
Could maybe be a warning as well, but I'm more concerned about the `[parsers.csv]` part..